styles/index.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+
+
+* {
+    margin: 0px;
+    padding: 0px;
+    color: white;
+  }
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+  body {
+  background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+    z-index: -1;
+  background: var(--color-darkblue);
+  background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  background-size: cover;
+  background-repeat: repeat;
+  background-position: center;
+   
+  }
+
+h1{
+  margin-left: 42%;
+}
+p{
+  margin-left: 32%;
+}
+  form {
+    width: 70%;
+    margin-top: 50px;
+    background: rgba(44, 62, 80, 0.95);
+    max-width: 50%;
+    max-height: 50%;
+    padding: 40px;
+    margin-left: 25%;
+  }
+h1 {
+  font-weight: 400;
+  line-height: 1.2;
```

Comment:
For a better structuring the codes you should not include the same tag selectors (h1) with different properties.

Bad practice: 

h1 {
       font-size: 34px;
      }

h1 {
       color: white:
     }


Good practice 

h1 {
     font-size:  34px;
      color: white;
 }

Please follow the above practice  to make your code much better and look professional. :slightly_smiling_face: